cmd/schedule.go

```diff
@@ -1,26 +1,188 @@
 package main
 
 import (
+	"errors"
+	"flag"
 	"fmt"
 	"os"
+	"regexp"
+	"strconv"
+	"syscall"
+	"time"
 
 	"github.com/phomer/scheduler/accounts"
 	"github.com/phomer/scheduler/comm"
+	"github.com/phomer/scheduler/jobs"
+	"github.com/phomer/scheduler/log"
+	"github.com/phomer/scheduler/sig"
 )
 
 func main() {
-	fmt.Println("schedule ", os.Args[1:])
-
 	// Read in the config parms
 	config := accounts.FindClientConfig()
 
 	// Parse the args
-	request := comm.NewRequest(os.Args)
+	request := ParseArgs(config.Username)
 
 	// Setup the SIGINT handler
+	sig.Initialize()
+	sig.Catch(syscall.SIGINT, StopStreaming)
 
 	// Send the request
-	comm.MakeRequest(config, request)
+	response := comm.MakeRequest(config, request)
 
 	// Stream the response back
+	comm.DisplayStream(response)
+}
+
+type CliCommand int
+
+var jobid CliCommand = 0
+var cli_cmd string
+
+var job_flags = map[string]bool{
+	"tail":   true,
+	"output": true,
+	"status": true,
+	"remove": true,
+}
+
+func ParseArgs(username string) *jobs.Request {
```

Comment:
Similar to my previous feedback, the code from here and below seems like implements a CLI args/flags parser which adds more complexity. Is it possible to use one of the standard ones instead?